retrieval/target.go

```diff
@@ -429,7 +429,7 @@ func (t *Target) scrape(appender storage.SampleAppender) (err error) {
 
 	req, err := http.NewRequest("GET", t.URL().String(), nil)
 	if err != nil {
-		panic(err)
+		return err
```

Comment:
Yes, the standard library apparently changed, causing a test to fail.

On Thu, Sep 10, 2015 at 12:14 PM Brian Brazil <notifications@github.com>
wrote:

> In retrieval/target.go
> <https://github.com/prometheus/prometheus/pull/1068#discussion_r39144188>:
>
> > @@ -429,7 +429,7 @@ func (t *Target) scrape(appender storage.SampleAppender) (err error) {
> >
> >  	req, err := http.NewRequest("GET", t.URL().String(), nil)
> >  	if err != nil {
> > -		panic(err)
> > +		return err
>
> Did you mean to include this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/1068/files#r39144188>.
>